recipes/qt/5.x.x/conandata.yml

```diff
@@ -1,4 +1,46 @@
 sources:
+  "5.15.16":
+    url:
+      - "https://download.qt.io/official_releases/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://download.qt.io/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://qt-mirror.dannhauer.de/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://mirror.netcologne.de/qtproject/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://ftp.fau.de/qtproject/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://ftp.nluug.nl/languages/qt/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://mirrors.dotsrc.org/qtproject/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://mirrors.20i.com/pub/qt.io/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://mirrors.ukfast.co.uk/sites/qt.io/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://ftp.acc.umu.se/mirror/qt.io/qtproject/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://www.nic.funet.fi/pub/mirrors/download.qt-project.org/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://qt.mirror.constant.com/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://mirrors.cloud.tencent.com/qt/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://mirror.bjtu.edu.cn/qt/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://mirrors.ustc.edu.cn/qtproject/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://ftp.jaist.ac.jp/pub/qtproject/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+      - "https://mirror.aarnet.edu.au/pub/qtproject/archive/qt/5.15/5.15.16/single/qt-everywhere-opensource-src-5.15.16.tar.xz"
+    sha256: "efa99827027782974356aceff8a52bd3d2a8a93a54dd0db4cca41b5e35f1041c"
+  "5.15.15":
```

Comment:
Thanks for adding the new versions!

Feel free to disregard 5.15.15 altogether, and even to stop publishing new revisions for 5.15.14 moving forwards
since https://github.com/conan-io/conan-center-index/pull/24275 we are only maintaining a single version for the 5.15.x branch, advising:
* Requiring with a version range `qt/[~5.15]` if one wishes to use the most recent patch release
* Using lockfiles (paired with a version range) if one wishes to stay on any particular one 
* Previously published versiones are *not* removed from the remote - but if there's a new patch release that will work (as per Qt backwards compatibility guarantees), then that's less incentive to keep older versions


(This is very specific to qt 5.15.x)

Reply:
thank you for the info! Removed 5.15.14 and 5.15.15